### PR TITLE
Suzie wednesday

### PR DESCRIPTION
**Pull request by suziestephen**

### CODE REVIEW

File: models/review.js [hunk 1]
```diff
@@ -32,9 +40,34 @@ module.exports = (sequelize, DataTypes) => {
...
-  return Post;
+{timestamps: false});
+
+
+  return Review;
 };
+
+
+  
+    // id: {
```
Comment (malakhim) [inline models/review.js]: Make sure you delete any unnecessary comments when creating a PR, this stuff shouldn't be making it into the main code

File: public/addreview.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h3 class="display-8">My Movie List</h3>
       <div class="col-md-5 offset-md-3">
         <form id="addreview">
           <div class="form-group">
-            <label for="title">Movie Title:</label>
+            <label for="title">Movie ID:</label>
```
Comment (malakhim) [inline public/addreview.html]: Should this be `title`?

File: routes/api-routes.js [hunk 3]
```diff
@@ -32,59 +35,69 @@ module.exports = (app) => {
...
-            source: req.body.source,
-            author: req.body.author,
-        }).then((dbPost) => res.json(dbPost));
-    });
+// // Routes
+// // =============================================================
+// module.exports = (app) => {
+  // GET route for getting all of the posts
+  app.get('/api/reviews/', (req, res) => {  // same format must be lower case, plural in api/ path 
+    db.Review.findAll({}).then((dbReview) => res.json(dbReview)); //Review is same format as variable Post in review.js in original excercise which was Post (capital P and singular)
```
Comment (malakhim) [inline routes/api-routes.js]: Don't put this in the code, this means nothing to someone reading your code after this pull request so just leave it as a comment in the pull request

Comment (suziestephen) [reply routes/api-routes.js]: Do you mean the //  comments not the app.get call?

File: public/addreview.html (hunk 2 above)
Comment (suziestephen) [reply public/addreview.html]: Its only the front end display text but yes it should. Should I change no or after the PR?

File: models/review.js (hunk 1 above)
Comment (suziestephen) [reply models/review.js]: Okay can do - again do I do this now or afte the PR?

File: public/addreview.html (hunk 2 above)
Comment (malakhim) [reply public/addreview.html]: Change now please, you should not have broken code (even if it's frontend display text) merged into main

File: routes/api-routes.js (hunk 3 above)
Comment (malakhim) [reply routes/api-routes.js]: Yup

**Review by malakhim — APPROVED** [review]
LGTM. Thanks for doing this

**Review by Criscel — APPROVED** [review]

**Pull request merged**